### PR TITLE
blog post: Your Container Volumes Served By oVirt

### PR DESCRIPTION
**Pull request by rgolangh**
New blog post on ovirt flex driver and provisioner

@mykaul please review (it's first draft)

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): (please @mention yourself to sign)

This pull request needs review by: (please @mention the reviewer if relevant)

### CODE REVIEW

**Comment by rgolangh** [comment]
review please @jasonbrooks @sandrobonazzola

File: source/blog/2018-02-20-Your-Container-Volumes-Served-By-oVirt.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+## Overview
+Consider this: you want to deploy wikimedia as a container, with all its content served from `/opt`. 
+For that you will create a persistent volume for the container - when we have state to keep and server
+creating a volume makes sense. It is persistent, it exists regardless the container state,
+and you can choose which directory exactly you serve that volume, and that is the most important
+part, k8s/openshift gives you an Api to determine who will provide the volume that you need.
+
+There are many options, cinder, aws, nfs and more. And in case the node that your pod is running on
+is underneath a VM in oVirt, you can use ovirt-flexdriver to attach an ovirt disk and that will
```
Comment (rollandf) [inline source/blog/2018-02-20-Your-Container-Volumes-Served-By-oVirt.md]: Please update all occurrences to oVirt

File: source/blog/2018-02-20-Your-Container-Volumes-Served-By-oVirt.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+title: Your Container Volumes Served By oVirt 
+author: rgolan
+tags: oVirt, oVirt 4.2, open source virtualization,kubernetes, openshift, external-storage, flexdriver, provisioner
+date: Tue Feb 20 08:54:34 IST 2018
+---
+
+> Note: _< 5 minutes read_
+
+When running a virtualization workload on oVirt, a VM disk is 'natively' a disk somewhere on your network-storage.  
+Entering containers world, on k8s or Openshift, there are many options specifically because the workload can be totally stateless, i.e
```
Comment (rollandf) [inline source/blog/2018-02-20-Your-Container-Volumes-Served-By-oVirt.md]: Openshift -> OpenShift
(there are more occurrences)

File: source/blog/2018-02-20-Your-Container-Volumes-Served-By-oVirt.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+READMORE
+
+## Overview
+Consider this: you want to deploy wikimedia as a container, with all its content served from `/opt`. 
+For that you will create a persistent volume for the container - when we have state to keep and server
+creating a volume makes sense. It is persistent, it exists regardless the container state,
+and you can choose which directory exactly you serve that volume, and that is the most important
+part, k8s/openshift gives you an Api to determine who will provide the volume that you need.
+
+There are many options, cinder, aws, nfs and more. And in case the node that your pod is running on
```
Comment (rollandf) [inline source/blog/2018-02-20-Your-Container-Volumes-Served-By-oVirt.md]: Cinder
AWS
NFS
API

File: source/blog/2018-02-20-Your-Container-Volumes-Served-By-oVirt.md (hunk 2 above)
Comment (rgolangh) [reply source/blog/2018-02-20-Your-Container-Volumes-Served-By-oVirt.md]: @rollandf uploaded new commit please check

Comment (mykaul) [reply source/blog/2018-02-20-Your-Container-Volumes-Served-By-oVirt.md]: At least once, you need to mention something like:
Kuberenetes (k8s), then you can use 'k8s' later on.

File: source/blog/2018-02-20-Your-Container-Volumes-Served-By-oVirt.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+In this post I want to cover how oVirt can provide volumes for containers running on k8s/openshift cluster.
+
+READMORE
+
+## Overview
+Consider this: you want to deploy wikimedia as a container, with all its content served from `/opt`. 
+For that you will create a persistent volume for the container - when we have state to keep and server
+creating a volume makes sense. It is persistent, it exists regardless the container state,
+and you can choose which directory exactly you serve that volume, and that is the most important
+part, k8s/openshift gives you an Api to determine who will provide the volume that you need.
```
Comment (mykaul) [inline source/blog/2018-02-20-Your-Container-Volumes-Served-By-oVirt.md]: Api -> API.
Also, in other places you use 'api'.

File: source/blog/2018-02-20-Your-Container-Volumes-Served-By-oVirt.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+
+There are many options, Cinder, AWS, NFS and more. And in case the node that your pod is running on
+is underneath a VM in oVirt, you can use ovirt-flexdriver to attach an oVirt disk and that will
```
Comment (mykaul) [inline source/blog/2018-02-20-Your-Container-Volumes-Served-By-oVirt.md]: underneath -> on

Comment (rgolangh) [reply source/blog/2018-02-20-Your-Container-Volumes-Served-By-oVirt.md]: done

File: source/blog/2018-02-20-Your-Container-Volumes-Served-By-oVirt.md (hunk 4 above)
Comment (rgolangh) [reply source/blog/2018-02-20-Your-Container-Volumes-Served-By-oVirt.md]: done

File: source/blog/2018-02-20-Your-Container-Volumes-Served-By-oVirt.md (hunk 2 above)
Comment (rgolangh) [reply source/blog/2018-02-20-Your-Container-Volumes-Served-By-oVirt.md]: done

**Pull request merged**